.github/workflows/branch_build.yml

```diff
@@ -55,6 +55,26 @@ jobs:
         echo "/usr/local/kubebuilder/bin" >> $GITHUB_PATH
     - name: Test
       run: make operator/test
+
+    - name: Build
```

Comment:
This part is failing, what is purpose of it, do we really need it?

Reply:
Yes, the first step is building, and then running WS scans. 
Reference: https://github.com/whitesource-ft/ws-examples/tree/main/CI-CD

Reply:
Looks like it's better now but still token should be updated 
```
Bad org token
```